### PR TITLE
🚩 add renewalEnabled feature flag

### PR DESCRIPTION
**Pull request by anncatton**
To disable renewal/expiry jobs and renew endpoint

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -102,4 +102,9 @@ DACO_ENCRYPTION_KEY=
...
 # Testing
 #############
-ADMIN_PAUSE=false
\ No newline at end of file
+ADMIN_PAUSE=false
+
+#############
+# Feature flags
+#############
+RENEWAL_FEATURE_ENABLED=false
```
Comment (joneubank) [inline .env.example:110]: Good practice is to have a consistent feature flag naming scheme like `FEATURE_RENEWAL_ENABLED`.

File: src/config.ts [hunk 2]
```diff
@@ -190,6 +193,9 @@ const buildAppContext = (): AppConfig => {
       },
     },
     adminPause: process.env.ADMIN_PAUSE === 'true' || false,
+    featureFlags: {
+      renewalEnabled: process.env.RENEWAL_FEATURE_ENABLED === 'true',
```
Comment (joneubank) [inline src/config.ts:197]: 👍 default false

File: src/jobs/runAllJobs.ts [hunk 3]
```diff
@@ -29,16 +39,18 @@ export default async function (
...
-    );
-    const secondExpiryNotificationReport = await secondExpiryNotificationCheck(
-      currentDate,
-      emailClient,
-    );
-    const expiringAppsReport = await runExpiringAppsCheck(currentDate, emailClient, user);
-    const closedRenewalsReport = await runCloseUnsubmittedRenewalsCheck(currentDate, user);
+    const firstExpiryNotificationReport = renewalEnabled
+      ? await firstExpiryNotificationCheck(currentDate, emailClient)
+      : getReportToBeImplemented('FIRST EXPIRY NOTIFICATIONS');
```
Comment (joneubank) [inline src/jobs/runAllJobs.ts:44]: report to be implemented is probably wrong for this, lets write something like `getReportFeatureDisabled`

**Review by joneubank — CHANGES_REQUESTED** [review]
Can you also add documentation somewhere (README is fine) that lists the feature flags in the application, how to enable them, and what they control?

File: .env.example (hunk 1 above)
Comment (anncatton) [reply .env.example:110]: aha i was wondering if there was a convention - will change!

Comment (joneubank) [reply .env.example:110]: Don't think there is a general convention, but throughout other argo apps this is how they're written so at least we're internally consistent.

**Review by joneubank — APPROVED** [review]

**Pull request merged**